src/MoBi.UI/Views/ValueEdit.cs

```diff
@@ -51,20 +52,27 @@ public override void InitializeBinding()
          tbValue.EnterMoveNextControl = true;
       }
 
-      public override bool HasError
+      private void onValueUpdating(ValueEditDTO dto, PropertyValueSetEventArgs<double> e)
       {
-         get { return _screenBinder.HasError; }
+         // Some artifact with this control when values have a lot of digits could lead to an event being raised even though nothing has changed.
+         // This should be fixed in OSPSuite.DataBinding but we'll be done at another time
+         if (ValueComparer.AreValuesEqual(e.OldValue, e.NewValue))
```

Comment:
@Yuri05 This is an issue with the DataBinding component. Only fixes this here for now and I created an issue in DataBinding to fix it later on